### DISCUSSION TITLE
Which generated files are should be kept in version control?

### DISCUSSION THREAD

**Discussion by lesomnus** [medz/dart-orm#441] [Q&A] [answered]
following files are generated after `npx prisma migrate dev`

```
/prisma/migrations/20241023155544_foo/migration.sql
/prisma/migrations/migration_lock.toml
/prism/dev.db-journal
/dmmf.json
```

also, can I rename `20241023155544_foo`? to something like `bar`?

**Comment by medz** [comment, marked as answer]
```text
/prism/dev.db-journal
/prisma/migrations/*
```

It looks like the dmmf.json was created due to an oversight on my part, I'll fix that.

**Reply by medz** [reply]
> also, can I rename 20241023155544_foo? to something like bar?

If you have already run the migration, then it is unsafe to rename it. It will cause the entire database to be reset.

Renaming the migration name will only affect the database that has already run the migration. If you are still using it for formal deployment, then you can rename it directly.
